src/tests/unit/pages/home/categories-list/category-item/category-item.spec.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import { render } from '@testing-library/react';
+import { useDispatch } from 'react-redux';
+import CategoryItem from '../../../../../../pages/home/categories-list/category-item/category-item';
+
+jest.mock('../../../../../../pages/home/categories-list/category-item/category-item.style', () => ({
+  useStyles: () => ({})
+}));
+jest.mock('react-redux');
+
+const mockDispatch = jest.fn();
+useDispatch.mockReturnValue(mockDispatch);
+
+describe('CategoryItem component tests', () => {
+  const mockProps = {
+    categoryName: 'testName',
+    categoryImageUrl: 'testImage',
+    categoryUrl: 'testUrl'
+  };
+
+  const { getByRole, getAllByText } = render(
+    <BrowserRouter>
+      <CategoryItem {...mockProps} />
+    </BrowserRouter>
+  );
+
+  test('Should render CategoryItem component', () => {
+    const linkElement = getByRole('link');
+    expect(linkElement.getAttribute('href')).toBe('/testUrl');
+
```

Comment:
What would you do if there is not only one link? Maybe try use {name: 'anchor'} ?

Reply:
Good remark.
Can you give a hint for beginner? 
What exactly can I do with links? 
Can I write a test-id attribute in the component?

Reply:
ok, that also could be solved with getByRole('link',{name: 'acncor_text_of_the_link'})

Reply:
Ok, thanks. 
I will learn such a way